pkg/render/manager.go

```diff
@@ -315,6 +317,26 @@ func (c *managerComponent) managerVolumes() []v1.Volume {
 					},
 				},
 			},
+			v1.Volume{
+				// We mount the full secret to be shared with Voltron.
+				Name: ManagerInternalTLSSecretName,
+				VolumeSource: v1.VolumeSource{
+					Secret: &v1.SecretVolumeSource{
+						SecretName: ManagerInternalTLSSecretName,
+					},
+				},
+			},
+		)
+
+		// Append volume for tunnel certificate
+		v = append(v, v1.Volume{
+			Name: VoltronTunnelSecretName,
+			VolumeSource: v1.VolumeSource{
+				Secret: &v1.SecretVolumeSource{
+					SecretName: VoltronTunnelSecretName,
+				},
+			},
+		},
```

Comment:
nit: just include this in the above `append`
```suggestion
			},
			v1.Volume{
				Name: VoltronTunnelSecretName,
				VolumeSource: v1.VolumeSource{
					Secret: &v1.SecretVolumeSource{
						SecretName: VoltronTunnelSecretName,
					},
				},
			},
```

Reply:
Sounds good :)